scripts/check_metadata_api.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Checks the compatbility of the Metadata API against the AMB specification
+
+source scripts/utils.sh
+
+TMP_DIR=$(mktemp -tdu serlo-database-layer-check-metadata.XXXXXXXXXX)
+ZIP_FILE="$TMP_DIR/amb.zip"
+SCHEMAS_PATH="amb-main/draft/schemas/"
+SCHEMAS_DIR="$TMP_DIR/$SCHEMAS_PATH"
+SCHEMA="${SCHEMAS_DIR}schema.json"
+
+function init() {
+  set -e
+  trap 'tear_down' EXIT
+
```

Comment:
> Usage Error: Couldn't find the node_modules state file - running an install might help (findPackageLocation)

Do we maybe have to add a `yarn install`?

Reply:
You have to run `yarn` in order to install all dependencies

Reply:
done!